### PR TITLE
Implement Allinc plugin

### LINKED ISSUE #61: Create plugin that loads all plugins
Create a new plugin in `plugins/allinc` with the package name `@pimd/allinc`. This should load all plugins you can find in `plugins/*-plugin` (in other words: all extensions to PIMD but not the Gulp integration). All plugins should be loaded with `config.use(insertPluginNameHere)`. 

The use case case for this: Lazy people can just start with all functionality available without caring.

### PR DESCRIPTION
**Pull request by hagenburger**
Resolves #61

### CODE REVIEW

File: plugins/allinc/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "author": "Nico Hagenburger <nico@hagenburger.net> (https://twitter.com/hagenburger)",
+  "keywords": [
+    "markdown",
+    "pimd-plugin"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "MIT",
+  "devDependencies": {
```
Comment (benevbright) [inline plugins/allinc/package.json]: Shouldn't it be `"dependencies"`?

File: plugins/allinc/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "@pimd/classes-plugin": "0.1.5",
+    "@pimd/highlight-plugin": "0.1.3",
+    "@pimd/html-injector-plugin": "0.1.1",
+    "@pimd/id-plugin": "0.1.4",
+    "@pimd/preview-plugin": "0.1.5",
+    "@pimd/prism-plugin": "0.1.1",
+    "@pimd/showmore-plugin": "0.1.1",
+    "pimd": "^0.5.0"
```
Comment (benevbright) [inline plugins/allinc/package.json]: I think `pimd` should be in devDependencies.

**Comment by benevbright** [comment]
All plugin works like a charm! 👍

**Comment by benevbright** [comment]
Just one thing, I'm not sure if including `prism-plugin` here is right. 
Of course, it's also one of the pimd plugins but it's too powerful and doing many things. 
Don't know if you think it's a problem or not. Just asking.

**Comment by hagenburger** [comment]
@benevbright this is to get started as easy as possible. People expect syntax highlighting in my opinion. If anybody does not agree, it’s still quite easy to load all plugins manually. I would also like to have more syntax highlighter ([Highlight.js](https://highlightjs.org) for example).

**Pull request merged**